libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -216,6 +232,12 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
     tZone: Array<GroupField>;
     bZone: Array<GroupField>;
```

Comment:
I do not see how these `zones` are used.
Do we still need it?

Reply:
Not needed. Removed